### PR TITLE
Update Openebs CSI driver feature support

### PR DESCRIPTION
**Pull request by prateekpandey14**
Signed-off-by: prateekpandey14 <prateekpandey14@gmail.com>

```release-note
NONE
```

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -8,57 +8,58 @@ The following are a set of CSI driver which can be used with Kubernetes:
 Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persistence (Beyond Pod Lifetime) | Supported Access Modes | Dynamic Provisioning | Other Features
 -----|-----------------|--------------------------------|-------------|-----------------------------------|------------------------|----------------------|--------
 [Alicloud Disk](https://github.com/AliyunContainerService/csi-plugin) | `diskplugin.csi.alibabacloud.com` | v1.0 | A Container Storage Interface (CSI) Driver for Alicloud Disk | Persistent | Read/Write Single Pod | Yes | Raw Block, Snapshot
-[Alicloud NAS](https://github.com/AliyunContainerService/csi-plugin) | `nasplugin.csi.alibabacloud.com` | v1.0 | A Container Storage Interface (CSI) Driver for Alicloud Network Attached Storage (NAS) | Persistent | Read/Write Multiple Pods | No |
```
Comment (saad-ali) [inline book/src/drivers.md]: Can you remove the spacing changes (or move them to a different PR).

Comment (prateekpandey14) [reply book/src/drivers.md]: Done
